packages/common/package.json

```diff
@@ -30,12 +30,15 @@
     "browser.js"
   ],
   "dependencies": {
+    "@eth-optimism/hardhat-ovm": "^0.2.2",
     "@ethersproject/bignumber": "^5.0.5",
     "@google-cloud/kms": "^0.3.0",
     "@google-cloud/storage": "^2.4.2",
     "@maticnetwork/maticjs": "^2.0.40",
+    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.10",
     "@nomiclabs/hardhat-etherscan": "^2.1.3",
     "@nomiclabs/hardhat-truffle5": "^2.0.0",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
```

Comment:
Do you see any reason why hardhat-related dependencies need to be exported in the package? Could we move them all down to dev dependencies. Waffle especially seems like a dev dependency since it's only used in tests.

I only mention this because every dependency we add here gets built into every frontend or node app that uses our contracts.

Reply:
I think we can move them to devDependencies...

Reply:
Done